ext/gosu/extconf.rb

```diff
@@ -101,9 +98,6 @@
   end
 
   pkg_config "sdl2"
-  pkg_config "vorbisfile"
-  pkg_config "sndfile"
-  pkg_config "libmpg123"
```

Comment:
Not sure why they were still in there, but now I'm worried I might have removed too many unused audio libraries form the "Getting Started on Linux" page 🤔